test.js

```diff
@@ -422,3 +422,54 @@ test('test that missing type errors', function (t, ctx, done) {
   ws.write(data)
   ws.end()
 })
+
+;[0, 1, 2, 10, 20, 100].forEach(function (max) {
+  test('test maxBufferLength: ' + max, testMaxBuffer(max, false))
+  test('test maxBufferLength: ' + max + ' (random)', testMaxBuffer(max, true))
+})
+
+function testMaxBuffer (max, randomize) {
+  return function (t, ctx, done) {
+    var ws = WriteStream(ctx.db, { maxBufferLength: max })
+    var sourceData = []
+    var batches = []
+
+    for (var i = 0; i < 20; i++) {
+      sourceData.push({ key: i < 10 ? '0' + i : String(i), value: 'value' })
```

Comment:
I guess this to make i < 10 sort below "10" right?

Reply:
Yes